conf/messages

```diff
@@ -392,6 +392,14 @@ authorisations.addAuthorisationsYesNo.checkYourAnswersLabel = Do you want to add
 authorisations.addAuthorisationsYesNo.change.hidden = if you want to add an authorisation
 authorisations.addAuthorisationsYesNo.error.required = Select yes if you want to add an authorisation
 
+
+authorisations.addArrivalDateYesNo.title = Do you want to add the arrival date at the office of destination?
+authorisations.addArrivalDateYesNo.heading = Do you want to add the arrival date at the office of destination?
+authorisations.addArrivalDateYesNo.paragraph = This is the latest date you expect the transit to arrive.
+authorisations.addArrivalDateYesNo.checkYourAnswersLabel = Do you want to add the arrival date at the office of destination?
+authorisations.addArrivalDateYesNo.change.hidden = if you want to add an arrival date
```

Comment:
Cool 👍 There is likely an answers helper however that we need to add a method to for this question. See https://github.com/hmrc/ctc-departure-transport-details-frontend/blob/094a198a500cf0bc3759f2353792e2d57acdeaa4/app/utils/cyaHelpers/TransportAnswersHelper.scala#L151